easybuild/easyconfigs/c/CPLEX/CPLEX-12.6.3.eb

```diff
@@ -0,0 +1,18 @@
+name = 'CPLEX'
+version = '12.6.3'
+
+homepage = 'http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/'
+description = """IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency,
+ reducing costs, and increasing profitability."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['cplex_studio%s.linux-x86-64.bin' % ''.join(version.split('.'))]
+
+dependencies = [
+    ('Java', '1.8.0_66', ''),
+    ('foss', '2015b', ''),
```

Comment:
In that case, using `foss` as a toolchain makes sense. The other alternative is to add `OpenMPI` as a dependency, and use `GCC` as a toolchain (since BLAS/LAPACK/FFTW is not relevant here).

For Java, just including it as a dep is enough imho.